roles/openshift_common/tasks/atomic_docker_check.yml

```diff
@@ -0,0 +1,16 @@
+---
+# Fail early if Atomic and old version of Docker
+- block:
+
+  - name: Determine available Docker
+    shell: "rpm -q --queryformat '---\ncurr_version: %{VERSION}\n' docker"
+    register: l_atomic_docker_version_result
+
+  - set_fact:
+      l_atomic_docker_version: "{{ l_atomic_docker_version_result.stdout | from_yaml }}"
+
+  - fail:
+      msg: Installation on Atomic Host requires Docker 1.12 or later. Please upgrade and restart the Atomic Host.
+    when: l_atomic_docker_version.curr_version | version_compare('1.12','<')
+
+  when: openshift.common.is_atomic | bool
```

Comment:
Should we be limiting this type of check only to AH installs? Wouldn't we also want to fail fast for other types of deployments if the version of docker installed (or available) does not meet the requirements as well?